.github/workflows/ci.yaml

```diff
@@ -0,0 +1,22 @@
+name: 'GitHub CI'
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x, 16.x]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: yarn install
+      - run: yarn lint
+      - run: yarn test:coverage
```

Comment:
should be enough to add https://github.com/codecov/codecov-action

Reply:
That's a good idea.

Reply:
#336